src/cdk-experimental/menu/menu.ts

```diff
@@ -26,8 +26,11 @@ import {Subject} from 'rxjs';
   },
 })
 export class CdkMenu implements OnDestroy {
-  /** Orientation of the menu - does not affect styling/layout. */
-  @Input('cdkMenuOrientation') _orientation: 'horizontal' | 'vertical' = 'vertical';
+  /**
+   * Orientation of the menu - does not affect styling/layout.
```

Comment:
You could drop this, now -- the rest is clear enough.

Reply:
(The 'orientation of the menu' bit -- 'does not affect styling/layout' is important. Git doesn't seem to have sub-line comments...)

Reply:
Sorry I don't follow. Which part should I remove?

Reply:
You can remove 'orientation of the menu'. It doesn't add anything at this point.